### ISSUE TITLE
Run in existing terminal panel

### ISSUE THREAD

**Issue by schrodog** [formulahendry/vscode-code-runner#417]
Right now code runner will open new terminal for execution. Is it possible to reuse existing terminal panel without opening new 'Code' terminal?

**Comment by anishmittal2020** [comment]
Yes, I too need this feature. It wastes some time loading bashrc unnecessarily.

**Comment by hamad12a** [comment]
It looks like no one cares about it!

**Comment by hamad12a** [comment]
A quick non-practical solution to this issue would be by editing the line `this._terminal = vscode.window.createTerminal("Code");` which comes included in 

```
private async executeCommandInTerminal(executor: string, appendFile: boolean = true) {
        let isNewTerminal = false;
        if (this._terminal === null) {
            this._terminal = vscode.window.createTerminal("Code");
            isNewTerminal = true;
        }
```
at **codeManager.ts** source file into  `this._terminal = vscode.window.activeTerminal;`; this allows to run the code on focused terminal.
